src/components/HelpMark/consts.ts

```diff
@@ -0,0 +1,8 @@
+import type {IconSize} from './HelpMark';
+
+export const ICON_SIZE_MAP: Record<IconSize, number> = {
```

Comment:
I think that it would be better to have `IconSize` to be inherited from this map, to have type updated after we change map

```
type IconSize = keyof typeof ICON_SIZE_MAP;
```

Reply:
Let's add the ability to set a custom size as number. I created issue for custom size and map doesn't have needed size)